pkg/debug/build.yml

```diff
@@ -2,13 +2,25 @@ image: eve-debug
 org: lfedge
 network: yes
 config:
+  pid: host
+  net: host
   binds:
+    - /:/hostfs
+    - /config:/config
+    - /persist:/persist:rshared,rbind
+    - /run:/run
+    - /tmp:/tmp
     - /dev:/dev
+    - /sys:/sys
+    - /containers:/containers
+    - /etc:/hostroot/etc
     - /etc/resolv.conf:/etc/resolv.conf
-    - /run:/run
-    - /var/log:/var/log/dom0
-    - /:/hostfs
-  net: host
+    - /etc/motd:/etc/motd
+    - /run:/root/.ssh
+    - /bin/eve:/bin/eve
+    - /usr/bin/ctr:/usr/bin/ctr
```

Comment:
I have wanted this for a while.

Reply:
In addition, can we set the env var `CONTAINERD_NAMESPACE=eve-user-apps`? That will make `ctr` commands default to using that namespace instead of `default`, where we don't run anything anyways.

We always can override it by doing, e.g. `ctr -n linuxkit.services ...` , but this way the default will be our user apps.

Reply:
Aha! Great idea! Will do.